fastai/callbacks/cpu_mem.py

```diff
@@ -0,0 +1,49 @@
+" Memory profiling callbacks "
+
+import tracemalloc, threading, torch, time
+from ..utils.mem import *
+from ..basic_train import *
+from ..torch_core import *
+
+class CpuPeakMemMetric(LearnerCallback):
+    "Callback that measures used and peaked general and CPU memory."
+
+    _order = -20  # Needs to run before the recorder
+
+    def __init__(self, learn: Learner):
```

Comment:
There is no need for an init if it only calls the super class.

Reply:
Hi @sgugger, I already fixed it. Thank you for your attention.